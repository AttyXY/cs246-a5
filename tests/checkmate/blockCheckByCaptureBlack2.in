setup
+ k e1
+ K e5
+ R a2
+ R b3
+ b h7
done
game human human
move b3 b1
quit
quit
