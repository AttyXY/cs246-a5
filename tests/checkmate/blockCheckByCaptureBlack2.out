WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  --------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  ----k---
   abcdefgh

8  --------
7  --------
6  --------
5  ----K---
4  --------
3  -R------
2  R-------
1  ----k---
   abcdefgh

8  --------
7  -------b
6  --------
5  ----K---
4  --------
3  -R------
2  R-------
1  ----k---
   abcdefgh

STARTING GAME!

8  --------
7  -------b
6  --------
5  ----K---
4  --------
3  -R------
2  R-------
1  ----k---
   abcdefgh

8  --------
7  -------b
6  --------
5  ----K---
4  --------
3  --------
2  R-------
1  -R--k---
   abcdefgh

Black is in check.
Final Score:
White: 0
Black: 0
THANKS FOR PLAYING CHESS!
